src/transform/EditTransform.js

```diff
@@ -64,11 +71,63 @@ const newEditSectionHeader = (document, index, level, titleHTML, showEditPencil
     element.appendChild(button)
   }
 
+  if (anchor && anchor.length > 0) {
+    // TODO: consider renaming this 'id' to 'anchor' for clarity - would need to update native
+    // code as well - used when TOC sections made to jump to sections.
+    element.id = anchor
+  }
+
   return element
 }
 
+/**
+ * Lead section header is a special case as it needs to show article title and description too,
+ * and in addition to the lead edit pencil, the description can also be editable.
+ * As a client, you may wish to set the ID attribute.
+ * @param {!Document} document
+ * @param {?string} articleDisplayTitle Article display title.
+ * @param {?string} titleDescription Article title description.
+ * @param {?string} addTitleDescriptionString Localized string e.g. 'Add title description'.
+ * @param {?boolean} isTitleDescriptionEditable Whether title description is editable.
+ * @param {?boolean} showEditPencil Whether to show the "edit" pencil (default is true).
+ * @param {?string} anchor Section anchor used for jumping between sections.
```

Comment:
Wouldn't this be '#'?

Reply:
checking...

Reply:
iirc - `id` from, say, mobile view, is just a number, which isn't specific enough to use because there could be cases where the section content could also contain an element with a numeric `id`. that's why mobile view also provides `anchor`, which should be more safe(r) to use for section container id (which is used for jumping to section).

Reply:
i suppose the parameter to be renamed... but i guess i thought it made more sense to leave it named the same as the api param...

Reply:
Removing this - on iOS we can add this after as-needed.

Reply:
Fixed! Good catch!